proto/axelar/reward/v1beta1/query.proto

```diff
@@ -9,11 +9,9 @@ import "axelar/reward/v1beta1/params.proto";
 option (gogoproto.goproto_getters_all) = false;
 
 // InflationRateRequest represents a message that queries the Axelar specific
-// inflation RPC method.
-message InflationRateRequest {
-  bytes validator = 1 [ (gogoproto.casttype) =
-                            "github.com/cosmos/cosmos-sdk/types.ValAddress" ];
-}
+// inflation RPC method. string is used for the validator address instead of
+// bytes so that it's supported as an arg in the generated REST API
```

Comment:
```suggestion
// inflation RPC method. Ideally, this would use ValAddress as the validator field type.
// However, this makes it awkward for REST-based calls, because it would expect a byte array as part of the url.
// So, the bech32 encoded string is used for this request instead.
```